.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Publishing of libs and plugin on release
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  publish:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
```

Comment:
If we don't have any specific reasons to build and release on JDK 17, I would switch to JDK 8. This way we ensure better compatibility with user code.

I'm planning downgrade kotlinx.fuzz from jdk 17 to jdk 8 as well in the next release

Reply:
Fixed